backend/src/LibExecution/ProgramTypesToDarkTypes.fs

```diff
@@ -987,6 +990,7 @@ module Const =
     | DEnum(_, _, [], "CBool", [ DBool b ]) -> PT.Const.CBool b
     | DEnum(_, _, [], "CString", [ DString s ]) -> PT.Const.CString s
     | DEnum(_, _, [], "CChar", [ DChar c ]) -> PT.Const.CChar c
+    | DEnum(_, _, [], "CChar", [ DString c ]) -> PT.Const.CChar c
```

Comment:
Should we remove the `| DEnum(_, _, [], "CChar", [ DChar c ]) -> PT.Const.CChar c` case?

Reply:
We should remove the `| DEnum(_, _, [], "CChar", [ DString c ]) -> PT.Const.CChar c` case, right?

Reply:
the `| DEnum(_, _, [], "CChar", [ DString c ])->...` is the one that works
in ProgramTypes we have `| CChar of String`

Reply:
ah yes you're right